getting error Unsupported URL Type "workspace:": workspace:*

**Describe the bug**
Attempting to install via npx as MCP, or direct global install, all result in error:

Unsupported URL Type "workspace:": workspace:*

**Troubleshooting Guide**
Try to follow the [Troubleshooting Guide](https://github.com/zilliztech/claude-context/blob/main/docs/troubleshooting/troubleshooting-guide.md) to solve the problem. If you can not solve the problem, please open an issue.

The noted troubleshoot guide is 404'ing

<img width="1206" height="507" alt="Image" src="https://github.com/user-attachments/assets/63ba1d46-4ba9-42f2-83e9-ba7174db94a1" />

attempted install log 
```
0 verbose cli /mnt/ddev-global-cache/n_prefix/pvcpipesupplies-web/bin/node /usr/local/bin/npm
1 info using npm@10.8.2
2 info using node@v20.17.0
3 silly config load:file:/mnt/ddev-global-cache/n_prefix/pvcpipesupplies-web/lib/node_modules/npm/npmrc
4 silly config load:file:/home/lucas/.npmrc
5 silly config load:file:/mnt/ddev-global-cache/n_prefix/pvcpipesupplies-web/etc/npmrc
6 verbose title npm install @pleaseai/context-please-mcp@latest
7 verbose argv "install" "--global" "@pleaseai/context-please-mcp@latest"
8 verbose logfile logs-max:10 dir:/mnt/ddev-global-cache/npm/_logs/2025-12-18T05_38_34_634Z-
9 verbose logfile /mnt/ddev-global-cache/npm/_logs/2025-12-18T05_38_34_634Z-debug-0.log
10 silly packumentCache heap:4345298944 maxSize:1086324736 maxEntrySize:543162368
11 silly logfile start cleaning logs, removing 1 files
12 silly logfile done cleaning log files
13 silly packumentCache corgi:https://registry.npmjs.org/@pleaseai%2fcontext-please-mcp cache-miss
14 http fetch GET 200 https://registry.npmjs.org/@pleaseai%2fcontext-please-mcp 1684ms (cache revalidated)
15 silly packumentCache corgi:https://registry.npmjs.org/@pleaseai%2fcontext-please-mcp set size:5180 disposed:false
16 silly idealTree buildDeps
17 silly fetch manifest @pleaseai/context-please-mcp@0.4.1
18 silly packumentCache full:https://registry.npmjs.org/@pleaseai%2fcontext-please-mcp cache-miss
19 http fetch GET 200 https://registry.npmjs.org/@pleaseai%2fcontext-please-mcp 1360ms (cache revalidated)
20 silly packumentCache full:https://registry.npmjs.org/@pleaseai%2fcontext-please-mcp set size:36354 disposed:false
21 silly placeDep ROOT @pleaseai/context-please-mcp@0.4.1 OK for:  want: 0.4.1
22 silly fetch manifest @modelcontextprotocol/sdk@^1.12.1
23 silly packumentCache full:https://registry.npmjs.org/@modelcontextprotocol%2fsdk cache-miss
24 verbose stack Error: Unsupported URL Type "workspace:": workspace:*
24 verbose stack     at unsupportedURLType (/mnt/ddev-global-cache/n_prefix/pvcpipesupplies-web/lib/node_modules/npm/node_modules/npm-package-arg/lib/npa.js:310:15)
24 verbose stack     at fromURL (/mnt/ddev-global-cache/n_prefix/pvcpipesupplies-web/lib/node_modules/npm/node_modules/npm-package-arg/lib/npa.js:367:13)
24 verbose stack     at Function.resolve (/mnt/ddev-global-cache/n_prefix/pvcpipesupplies-web/lib/node_modules/npm/node_modules/npm-package-arg/lib/npa.js:83:12)
24 verbose stack     at /mnt/ddev-global-cache/n_prefix/pvcpipesupplies-web/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:1008:39
File /mnt/ddev-global-cache/npm/_logs/2025-12-18T05_38_34_634Z-debug-0.log not changed so no update needed
```

Not an expert on things related to npm/npx, so i asked claude. he hays your latest release is broken

```
What the Error Means                                                                                                                                                                                             
                                                                                                                                                                                                                   
  The error EUNSUPPORTEDPROTOCOL - Unsupported URL Type "workspace:": workspace:* indicates that npm is encountering a dependency reference it doesn't understand.                                                 
                                                                                                                                                                                                                   
  The Root Cause:                                                                                                                                                                                                  
  - The package @pleaseai/context-please-mcp@0.4.1 was published to npm with a dependency that uses workspace:* as the version specifier                                                                           
  - The workspace: protocol is specific to pnpm and Yarn workspaces for managing monorepo packages                                                                                                                 
  - npm doesn't support this protocol - it expects actual version numbers or npm registry URLs                                                                                                                     
                                                                                                                                                                                                                   
  Is This a Local Environment Issue?                                                                                                                                                                               
  No, this is NOT a local environment issue. This is a bug in the published package itself. The package maintainers accidentally published version 0.4.1 with workspace references that should have been resolved  
  to actual version numbers before publishing.                                                                                                                                                                     
                                                                
```

but seems doe snot support OLLama

```
[LOG] [EMBEDDING] Creating ollama embedding instance...
2025-12-18T06:21:26.494Z [ERROR] MCP server "context-please" Server stderr: [EMBEDDING] ❌ Unsupported embedding provider: ollama
Fatal error: Error: Unsupported embedding provider: ollama
    at createEmbeddingInstance (file:///mnt/ddev-global-cache/npm/_npx/335d023a69445aa8/node_modules/@pleaseai/context-please-mcp/dist/embedding.js:63:19)
    at new ContextMcpServer (file:///mnt/ddev-global-cache/npm/_npx/335d023a69445aa8/node_modules/@pleaseai/context-please-mcp/dist/index.js:37:27)
    at main (file:///mnt/ddev-global-cache/npm/_npx/335d023a69445aa8/node_modules/@pleaseai/context-please-mcp/dist/index.js:293:20)
    at file:///mnt/ddev-global-cache/npm/_npx/335d023a69445aa8/node_modules/@pleaseai/context-please-mcp/dist/index.js:306:1
    at ModuleJob.run (node:internal/modules/esm/module_job:234:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:473:24)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)
```

So maybe need latest?

--- a/.github/workflows/release-please.yml
+++ b/.github/workflows/release-please.yml
@@ -2,6 +2,13 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
+    inputs:
+      publish_only:
+        description: 'Skip release-please and only publish packages'
+        required: false
+        default: 'false'
+        type: boolean
 
 permissions:
   contents: write
@@ -12,6 +19,7 @@ name: release-please
 jobs:
   release-please:
     runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' || inputs.publish_only != true }}
     outputs:
       releases_created: ${{ steps.release.outputs.releases_created }}
       core_released: ${{ steps.release.outputs['packages/core--release_created'] }}
@@ -31,7 +39,7 @@ jobs:
 
   publish:
     needs: release-please
-    if: ${{ needs.release-please.outputs.releases_created == 'true' }}
+    if: ${{ always() && (needs.release-please.outputs.releases_created == 'true' || inputs.publish_only == true) }}
     runs-on: ubuntu-latest
     environment: npm
     permissions:
@@ -49,7 +57,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '22.x'
+          node-version: '24.x'
           cache: 'pnpm'
           registry-url: 'https://registry.npmjs.org'
 
@@ -59,12 +67,13 @@ jobs:
       - name: Build packages
         run: pnpm build
 
+      # IMPORTANT: Use pnpm publish (not npm publish) to resolve workspace: protocols
+      # pnpm automatically converts workspace:* to actual version numbers during publish
+      # See: https://pnpm.io/cli/publish and https://github.com/pnpm/pnpm/issues/9812
       - name: Publish core package to npm
-        if: ${{ needs.release-please.outputs.core_released == 'true' }}
-        working-directory: packages/core
-        run: npm publish --provenance --access public
+        if: ${{ needs.release-please.outputs.core_released == 'true' || inputs.publish_only == true }}
+        run: pnpm --filter @pleaseai/context-please-core publish --access public --no-git-checks
 
       - name: Publish mcp package to npm
-        if: ${{ needs.release-please.outputs.mcp_released == 'true' }}
-        working-directory: packages/mcp
-        run: npm publish --provenance --access public
+        if: ${{ needs.release-please.outputs.mcp_released == 'true' || inputs.publish_only == true }}
+        run: pnpm --filter @pleaseai/context-please-mcp publish --access public --no-git-checks

